src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.swr

```diff
@@ -8,10 +8,10 @@ vs.dependencies
   vs.dependency id=Microsoft.Net.4.6.1.FullRedist.Resources
 
 folder InstallDir:\MSBuild\15.0\Bin
-    file source=$(OutputPath)\VBCSCompiler.exe vs.file.ngen=yes
```

Comment:
What is the point of `ngen=yes` if it's really `ngen=maybe`?  

Have we given feedback to the developers here on the time spent tracking down this problem?